### ISSUE TITLE
Markdown section awaiting review

### ISSUE THREAD

**Issue by zachary-foster** [grunwaldlab/Reproducible-science-in-R#2]
I am done with the content of the Markdown section and will wait until I get feedback before working on it more. The styling can be modified using the functions in `function.R` and the css in `tutorial.css`. Feel free to make changes or respond with comments on this issue. Once everyone feels it is ready, we can close the issue. Thanks!

**Comment by grunwald** [comment]
I finished my review. This looks great. I pushed my edits.

Question: Should we have one place for glossary or leave it at the end of Zach's doc?

Nice work!

**Comment by zkamvar** [comment]
Hi Nik,

This is the last commit we see from you: https://github.com/grunwaldlab/Reproducible-science-in-R/commit/6b328a29df7de61f008cbc171d07aea290ec82b3

Were these your edits?

**Comment by grunwald** [comment]
No. There should have been more … let me see what is going on.

Nik

**Comment by grunwald** [comment]
Aha. These were silly, test edits from our meetings, I undid. Just pushed edits for Markdown primer.

**Comment by zachary-foster** [comment]
Thanks! I like your edits. 

In regards to the glossary question:
The way that it is implemented now, the function that handles the glossary cannot compile terms from multiple rmd files, however I could make something work if we wanted to do that. 
I considered combining everything into a single glossary, but I thought it better (and easier) to put one on each page instead, since people might read a short glossary (~7 terms) as they read the whole document whereas they would likely skip a larger one, assuming it only practical as a reference. If they read the glossary, it should reinforce the important terminology in their memory. The idea is to trick them into reading many short glossaries, the sum of which they would ignore when presented together.

**Comment by zkamvar** [comment]
> Question: Should we have one place for glossary or leave it at the end of Zach's doc?

Why not both? Repetition is important for recollection.

**Comment by grunwald** [comment]
I like both, too. Let's go with that for now.

Nik

**Comment by zachary-foster** [comment]
I like the idea of doing both as well.

**Comment by zachary-foster** [comment]
I will make a new issue in regards to a cumulative bibliography.
